main_page.html

```diff
@@ -6,46 +6,59 @@
 	<title>Main Page</title>
 	<link rel="stylesheet" type="text/css" href="css/upls_styles.css">
 	<script src="js/main_menu.js"></script>
+	<script src="js/main_page.js"></script>
```

Comment:
js/main_page.js should be the last included file, because the api stuff needs to be included in order for signin stuff to work